yarn.lock

```diff
@@ -15047,7 +15047,7 @@ react-portal@^4.1.5:
   dependencies:
     prop-types "^15.5.8"
 
-react-redux@^5.0.7, react-redux@^5.1.0:
+react-redux@^5.0.7:
```

Comment:
why do we have both v5 and v7?

Reply:
Looks like both react-beautiful-dnd and d2-ui-favorites-dialog use it